src/debug/debugUtils.ts

```diff
@@ -57,31 +57,40 @@ export async function promptForAppPort(ports: string[], defaultPort: string, env
  * @param container the container name.
  * @return the commands associated with the processes.
  */
-export async function getCommandsOfProcesses(kubectl: Kubectl, pod: string, podNamespace: string | undefined, container: string): Promise<string[]> {
-    const commandLines: string[] = [];
-    const nsarg = podNamespace ? `--namespace ${podNamespace}` : '';
+export async function getProcesses(kubectl: Kubectl, pod: string, podNamespace: string | undefined, container: string | undefined): Promise<ProcessInfo[] | undefined> {
+    const processes: ProcessInfo[] = [];
+    const nsarg = podNamespace && podNamespace !== "default" ? `--namespace ${podNamespace}` : '';

```

Comment:
Is this additional check required?  Feels like it could go wrong if the `kubectl` context namespace was something other than `default`, and the user then tried to debug something in the `default` namespace.  Whereas in the 'normal' case the worst that would happen is that we have a redundant `--namespace default` on the generated `kubectl` command line.  I might be missing something though...?

Reply:
I may have added that thinking that a specific namespace name was required rather than the term `default`.  I have removed the check.